utilities/markChoreDone.js

```diff
@@ -1,26 +1,83 @@
+//node packages
+const {google} = require("googleapis");
+require("dotenv").config();
+
 //local packages
 const {
-  app: {
-    client: {
-      chat: { update }
+    app: {
+        client: {
+            chat: { update }
+        }
     }
-  }
 } = require("./bolt.js");
 
 //globals
 const TOKEN = process.env.SLACK_BOT_TOKEN;
+const privatekey = require("../keys/sheets-api.json");
+const SPREADSHEET_ID = process.env.SPREADSHEET_ID;
+const GDRIVE_EMAIL = process.env.GDRIVE_EMAIL;
+
+const jwtClient = new google.auth.JWT(
+    privatekey.client_email,
+    null,
+    privatekey.private_key,
+    ["https://www.googleapis.com/auth/spreadsheets"],
+    GDRIVE_EMAIL
+);
+
+const markChoreUser = async (user, date, choretext) => {
```

Comment:
Can we rename this to something a little more semantic?

Reply:
Would markChoreDoneSpreadsheet() work?

Reply:
How about, ummm...`giveCreditToUser` or something haha

Reply:
Actually, how about `setCompletedBy()`

Reply:
Resolved